packages/core/upload/admin/src/utils/updateUrlForCaching.js

```diff
@@ -0,0 +1,15 @@
+import { URL } from 'url';
+
+const updateUrlForCaching = (urlString, updatedAt) => {
```

Comment:
this needs some JSDoc

Reply:
Also nitpick, but i think the name could be better, what about making it more generic e.g. `appendSearchParamsToUrl` the type could be:

```ts
const appendSearchParamsToUrl = (url?: string, params: Record<string, string | number>) => string
```

You could then just do `Object.entries(params).forEach(([key, value]) => /* update URL */)` and it has more ability...

Reply:
I made this change since it didn't take much effort, but it feels a bit over-engineered for the only use case that currently needs it. There might be a better home for it where it would get more use, such as the utils package. Moving it at this point is more work than I'd like to do, unless there are known cases in other packages that could use it right away.